keyboards/lets_split/config.h

```diff
@@ -21,4 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
+#undef RGBLED_NUM
```

Comment:
This should be in your keymap's config.h, NOT the keyboards.

If you could remove this from this file, and add it to your keymap's config.h (eg, `/keyboards/lets_split/keymaps/pyrol/config.h`)